src/protocols/bitcoin/libs/BitcoinAdapter.ts

```diff
@@ -90,19 +103,22 @@ export class BitcoinAdapter extends BaseProtocolAdapter {
     seed: Uint8Array,
     accountIndex: number,
   ): IHdWalletAccount {
+    const { activeNetwork } = useNetworks();
+
+    const network = networks[activeNetwork.value.type as keyof typeof networks] || networks.bitcoin;
+    const pathCoinType = activeNetwork.value.type === NETWORK_TYPE_TESTNET ? 1 : 0;
+
     const node = this.bip32.fromSeed(Buffer.from(seed));
-    const path = `m/84'/0'/${accountIndex}'/0/0`; // 44 for Legacy
+    const path = `m/84'/${pathCoinType}'/${accountIndex}'/0/0`; // 44 for Legacy
```

Comment:
The comment says `44` but I don't see any `44` here. Are you sure this is correct?

Reply:
Added clear comments.